src/github.com/getlantern/flashlight/proxied/proxied.go

```diff
@@ -387,6 +387,10 @@ func chained(rootCA string, persistent bool) (http.RoundTripper, error) {
 			ClientSessionCache: clientSessionCache,
 		},
 	}
+	if persistent {
```

Comment:
It seems like this might make sense to add whether or not the connection is specified as persistent @oxtoacart. I believe `DisableKeepAlives` essentially specifies whether or not to include the `Connection: close` header, but it's also always possible some server or client along the line doesn't work like it's supposed to.